src/clients/openapi.js

```diff
@@ -94,7 +99,17 @@ export function http(req, serviceName, operationName) {
         return axios(
             request,
         ).then((response) => {
-            validateResponse(response);
+            const [validResponse, message] = validateResponse(response);
+            if (!validResponse) {
+                if (logFailure) {
+                    const error = {
```

Comment:
Note: we don't log the response body (unlike failed responses) - we don't want to log maintenance html pages.